api/src/pcapi/routes/backoffice_v3/offers/forms.py

```diff
@@ -0,0 +1,256 @@
+    CATEGORY = "Catégorie"
+    CREATION = "Date de création"
+    DEPARTMENT = "Département"
+    EAN = "EAN-13"
+    VALIDATION = "État"
+    ID = "ID de l'offre"
+    VENUE = "Lieu"
+    NAME = "Nom de l'offre"
+    OFFERER = "Structure"
+    TAG = "Tag"
+    VISA = "Visa d'exploitation"
+
+
+form_field_configuration = {
+    "CATEGORY": {"field": "category", "operator": ["IN", "NOT_IN"]},
+    "CREATION": {
+        "field": "date",
+        "operator": [
+            "GREATER_THAN_OR_EQUAL_TO",
+            "LESS_THAN",
+        ],
+    },
+    "DEPARTMENT": {"field": "department", "operator": ["IN", "NOT_IN"]},
+    "EAN": {"field": "string", "operator": ["CONTAINS", "NO_CONTAINS", "EQUALS", "NOT_EQUALS"]},
+    "ID": {"field": "integer", "operator": ["EQUALS", "NOT_EQUALS"]},
+    "NAME": {"field": "string", "operator": ["CONTAINS", "NO_CONTAINS", "EQUALS", "NOT_EQUALS"]},
+    "OFFERER": {"field": "offerer", "operator": ["IN", "NOT_IN"]},
+    "TAG": {"field": "criteria", "operator": ["IN", "NOT_IN"]},
+    "VENUE": {"field": "venue", "operator": ["IN", "NOT_IN"]},
+    "VALIDATION": {"field": "validation", "operator": ["IN", "NOT_IN"]},
+    "VISA": {"field": "string", "operator": ["CONTAINS", "NO_CONTAINS", "EQUALS", "NOT_EQUALS"]},
+}
+
+
+class GetOffersBaseFields(utils.PCForm):
+    sort = wtforms.HiddenField(
+        "sort", validators=(wtforms.validators.Optional(), wtforms.validators.AnyOf(("id", "dateCreated")))
+    )
+    order = wtforms.HiddenField(
+        "order", default="asc", validators=(wtforms.validators.Optional(), wtforms.validators.AnyOf(("asc", "desc")))
+    )
+    limit = fields.PCSelectField(
+        "Nombre maximum de résultats",
+        choices=((100, "100"), (500, "500"), (1000, "1000"), (3000, "3000")),
+        default="100",
+        coerce=int,
+        validators=(wtforms.validators.Optional(),),
+    )
+
+    def is_empty(self) -> bool:
+        # needed to clean multiple inheritance
+        return True
+
+
+class OfferAdvancedSearchSubForm(utils.PCForm):
+    class Meta:
+        csrf = False
+
+    json_data = json.dumps(
+        {
+            "display_configuration": form_field_configuration,
+            "all_available_fields": [
+                "category",
+                "criteria",
+                "date",
+                "department",
+                "integer",
+                "offerer",
+                "string",
+                "venue",
+                "validation",
+            ],
+            "sub_rule_type_field_name": "search_field",
+            "operator_field_name": "operator",
+        }
+    )
+
+    def __init__(self, *args: list, **kwargs: dict):
+        super().__init__(*args, **kwargs)
+        autocomplete.prefill_criteria_choices(self.criteria)
+        autocomplete.prefill_offerers_choices(self.offerer)
+        autocomplete.prefill_venues_choices(self.venue)
+
+    search_field = fields.PCSelectWithPlaceholderValueField(
+        "champ de recherche",
```

Comment:
```suggestion
        "Champ de recherche",
```
? Mais je n'ai pas le visuel en tête